src/address.js

```diff
@@ -1,11 +1,49 @@
 var Buffer = require('safe-buffer').Buffer
 var bech32 = require('bech32')
 var bs58check = require('bs58check')
+var bcrypto = require('./crypto')
 var bscript = require('./script')
 var networks = require('./networks')
 var typeforce = require('typeforce')
 var types = require('./types')
 
+// TODO: refactor
+function p2pkhFromHash (hash, network) {
+  network = network || networks.bitcoin
+  return toBase58Check(hash, network.pubKeyHash)
+}
+
+function p2pkhFromKeyPair (keyPair) {
```

Comment:
Would be nice if all the FromKeyPair functions could be called from the ECPair prototype.

keyPair.getAddress('p2wpkh') or maybe just a separate function?